packages/ui/src/components/PreviewItem/PreviewItem.tsx

```diff
@@ -0,0 +1,66 @@
+import MagnifyingGlassPlusIcon from "@heroicons/react/24/outline/MagnifyingGlassPlusIcon";
+import { ReactNode } from "react";
+import { CardBasic } from "../Card";
+import Button from "../Button";
+
+export interface PreviewItemProps {
+  title: string;
+  details: ReactNode[];
+  buttonName: string;
+}
+
+const PreviewItem = ({ title, details, buttonName }: PreviewItemProps) => {
+  return (
+    <div
+      data-h2-position="base(relative)"
+      data-h2-display="base(grid)"
+      data-h2-grid-template-columns="base(3fr 1fr)"
+      data-h2-align-items="base(center)"
+      data-h2-background-color="base(foreground)"
+      data-h2-padding="base(x1)"
+    >
+      <div>
+        <p
+          data-h2-font-weight="base(bold)"
+          data-h2-text-decoration="base(underline)"
+          data-h2-margin-bottom="base(x.5)"
+        >
+          {title}
+        </p>
+        <div
+          data-h2-display="base(flex)"
```

Comment:
This looks pretty rough in mobile mode.  I'm guessing @JoshBeveridge is going to say stack the items up for mobile.
![image](https://github.com/user-attachments/assets/299bc130-8a3a-4a0a-bf26-a5580570b657)

Reply:
I'm sure how the bullets would behave if that's the case though.

Reply:
Looks like Josh has a design for this now!  :partying_face: 
![image](https://github.com/user-attachments/assets/6e18b3b9-bf34-4ace-ab9d-53f6137f4f03)

Reply:
Yup! I'll have the changes up soon with the new design :smile: